imaging/de/java/adjust/emf/_index.md

```diff
@@ -24,25 +24,25 @@ und installieren Sie es in Ihrem Maven-basierten Projekt, indem Sie der pom.xml
 
 {{% blocks/products/pf/agp/code-block title="Repository" offSpacer="true" %}}
 
-„xml
-<Speicher>
-<id>AsposeJavaAPI</id>
-<name>Aspose-Java-API</name>
+```xml
+<repository>
+<id>Aspose Java API</id>
+<name>Aspose Java API</name>
 <url>https://repository.aspose.com/repo/</url>
-</Repository>
+</repository>
 ```
 
 {{% /blocks/products/pf/agp/code-block %}}
 
 {{% blocks/products/pf/agp/code-block title="Abhängigkeit" offSpacer="true" %}}
 
-„xml
-<Abhängigkeit>
+```xml
+<dependency>
 <groupId>com.aspose</groupId>
 <artifactId>aspose-imaging</artifactId>
-<version>Version der aspose-imaging API</version>
+<version>version of aspose-imaging API</version>
```

Comment:
Is it expected change ?

Reply:
@aifeigin yes, install package should be common.

Reply:
It means all package install text will be in english as common solution. You can see as done in other products - https://products-qa.aspose.com/cells/de/java/conversion/xls-to-xlsx/